torch/utils/cpp_extension.py

```diff
@@ -1572,7 +1597,12 @@ def _run_ninja_build(build_directory: str, verbose: bool, error_prefix: str) ->
         raise RuntimeError(message) from e
 
 
-def _import_module_from_library(module_name, path, is_python_module):
+def _import_module_from_library(module_name, path, is_python_module, is_standalone):
+    if is_standalone:
+        if IS_WINDOWS and TORCH_LIB_PATH not in os.getenv('PATH', ''):
```

Comment:
Technically this is wrong, because if I have `f'{TORCH_LIB_PATH}/blah/blah` in PATH it will incorrectly detect that the PATH is present when it actually isn't. I also get a bit nervous about forward slash and backward slash confusion here.

Reply:
Good point. I added `.split(';')` so now it should actually do the right thing. There shouldn't be any forward-backward slash shenaniganry since I use `os.path.blah` for everything. (Right?)

Reply:
It will be strictly safe if you may use `os.path.samefile` and it automatically handles cases and slashes. e.g.
```py
>>> os.path.samefile('C:\\Windows', 'c:/windows')
True
```